README.md

```diff
@@ -428,6 +428,53 @@ the [Twilio documentation](https://www.twilio.com/docs/libraries/go/usage-guide#
 
 ## Advanced Usage
 
+### Using Request Validator
+
+Validating GET/POST Requests are coming from Twilio:
+```go
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/twilio/twilio-go/client"
+)
+
+func main() {
+	// For this example: TWILIO_AUTH_TOKEN := "12345"
```

Comment:
Don't see the significance of this comment, should rather change to:
"You can find your Auth Token at twilio.com/console"

Reply:
For the validation in Line 466/474 to work (to return True), the Auth_Token must be 12345. I could add in another line to include where they can find their AuthToken